src/Illuminate/Routing/Router.php

```diff
@@ -769,6 +770,8 @@ public static function toResponse($request, $response)
             $response = (new HttpFoundationFactory)->createResponse($response);
         } elseif ($response instanceof Model && $response->wasRecentlyCreated) {
             $response = new JsonResponse($response, 201);
+        } elseif ($response instanceof Stringable) {
+            $response = new Response($response->__toString(), 200, ['Content-Type' => 'text/html']);
```

Comment:
Do we want this to take precedence over json?

Reply:
Yes, it needs to, otherwise the `$response instanceof JsonSerializable` check will cause the `Stringable` to be JSON encoded, which is exactly what we're trying to avoid.